setup.sh

```diff
@@ -20,35 +20,36 @@ osx() {
 
 install_apt() {
     sudo apt-get update || true
-    sudo apt-get -y install git gdb python3-dev python3-pip python3-setuptools libglib2.0-dev libc6-dbg
+    sudo apt-get install -y git gdb python3-dev python3-pip python3-setuptools libglib2.0-dev libc6-dbg nasm
+    test -f /usr/bin/go || sudo apt-get install -y golang
 
     if uname -m | grep x86_64 > /dev/null; then
         sudo dpkg --add-architecture i386 || true
         sudo apt-get update || true
-        sudo apt-get -y install libc6-dbg:i386 || true
+        sudo apt-get install -y libc6-dbg:i386 || true
     fi
 }
 
 install_dnf() {
     sudo dnf update || true
-    sudo dnf -y install gdb gdb-gdbserver python-devel python3-devel python-pip python3-pip glib2-devel make
+    sudo dnf -y install gdb gdb-gdbserver python-devel python3-devel python-pip python3-pip glib2-devel make nasm golang
```

Comment:
Since the developer can run tests and have all the set up, install `nasm` and `go` needs to be at this time, in the set up.

Reply:
Solved in this replay: https://github.com/pwndbg/pwndbg/pull/868#discussion_r549089436